blob/service.go

```diff
@@ -137,9 +138,8 @@ func (s *Service) Get(
 	return
 }
 
-// GetProof retrieves all blobs in the given namespaces at the given height by commitment
-// and returns their Proof. It collects all namespaced data from the EDS, constructs blobs
-// and compares commitments.
+// GetProof returns an NMT inclusion proof for a blob within the specified namespace at the given
```

Comment:
Let's also specify that the Proof itself is not a Blob inclusion Proof but the namespace inclusion proof to the respective row roots.

Reply:
@vgonkivs can you draft a suggested edit?

Reply:
`// GetProof returns an NMT inclusion proof for a specified namespace to the respective row roots on which the blob spans on`